jujugui/static/gui/src/app/init/test-hotkeys.js

```diff
@@ -0,0 +1,72 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2012-2013 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+
+const Keysim = require('keysim');
+const hotkeys = require('./hotkeys');
+const utils = require('../../test/utils');
+
+describe('hotkeys', function() {
+  let container, context, listener;
+  const keyboard = Keysim.Keyboard.US_ENGLISH;
+
+  beforeEach(() => {
+    context = {
+      _clearSettingsModal: sinon.stub(),
+      _clearShortcutsModal: sinon.stub(),
+      _displaySettingsModal: sinon.stub(),
+      _displayShortcutsModal: sinon.stub()
+    };
+    listener = hotkeys.activate(context);
+    container = utils.makeAppContainer();
+  });
+
+  afterEach(() => {
+    listener.deactivate();
+    container.remove();
```

Comment:
maybe `listener`, `context`, and `container` should be set to a default value here too? `undefined` or `null`?